enterprise/server/scheduling/priority_task_scheduler/priority_task_scheduler.go

```diff
@@ -557,6 +615,22 @@ func (q *PriorityTaskScheduler) Start() error {
 			q.handleTask()
 		}
 	}()
+
+	if *queueTrimInterval > 0 {
+		go func() {
+			timer := q.clock.NewTicker(*queueTrimInterval)
```

Comment:
nit: call this `ticker` (avoids confusion since `time.Timer` is also a thing)

Reply:
Done